source/common/json/config_schemas.cc

```diff
@@ -1436,6 +1436,18 @@ const std::string Json::Schema::CLUSTER_SCHEMA(R"EOF(
         "type" : "string",
         "enum" : ["round_robin", "least_request", "random", "ring_hash", "original_dst_lb"]
       },
+      "ring_hash_lb_config" : {
```

Comment:
Does new config need to go into the old v1 schema?  @htuch ?

Reply:
In this case it's needed for migration purposes.

Reply:
We should add to v1 docs also.